### DISCUSSION TITLE
How to create model name Shrimp

### DISCUSSION THREAD

**Discussion by luuhung93** [keystonejs/keystone#7550] [Questions] [answered]
`
export const lists: Lists = {
    ...
    Shrimp: list({
        fields: {
           name: text(),
        },
      }),
}
`

Error: Unable to use Shrimp as a List name - it has an ambiguous plural (Shrimp). Please choose another name for your list.

**Comment by jezzzm** [comment]
```export const lists: Lists = { ... Prawn: list({ fields: { name: text(), }, }), }```

**Comment by dcousens** [comment]
Duplicate of https://github.com/keystonejs/keystone/issues/7385,  I think this is technically a feature request but it's problematic that I think we can leave it as an issue

**Comment by dcousens** [comment, marked as answer]
You can now use `graphql.plural` to provide an unambiguous plural _for only the GraphQL API_.
The addition of configuration for `ui.plural` now supports (and defaults to) allowing ambiguous plurals in the UI.
- #7657

Example
```typescript
export const lists = {
  Shrimp: list({
    fields: {
      name: text(),
    },
    graphql: {
      plural: 'Shrimps'
    },
    // ui: { // un-comment to change the default plural from `Shrimp` in the admin interface
    //   plural: 'Shrimps' 
    // }
  })
}
```
